projects/admin/src/app/routes/items-route.ts

```diff
@@ -134,8 +134,10 @@ export class ItemsRoute extends BaseRoute implements RouteInterface {
             exportFormats: [
               {
                 label: 'CSV',
-                format: 'csv'
-              }
+                format: 'csv',
+                endpoint: this._routeToolService.apiService.endpointPrefix + '/item/inventory',
```

Comment:
`this._routeToolService.apiService.getEndpointByType('item/inventory')` ?

Reply:
ok, done